ui/components/Policies/PolicyList/PolicyTable.tsx

```diff
@@ -0,0 +1,127 @@
+import React from "react";
+
+import moment from "moment";
+import { FC } from "react";
+
+import { useFeatureFlags } from "../../../hooks/featureflags";
+import { formatURL } from "../../../lib/nav";
+import { V2Routes } from "../../../lib/types";
+import DataTable, { filterConfig } from "../../DataTable";
+import Link from "../../Link";
+import Text from "../../Text";
+import PolicyMode from "../Utilis/PolicyMode";
+import Severity from "../Utilis/Severity";
+import { Policy } from "../../../lib/api/core/core.pb";
+
+interface CustomPolicy extends Policy {
+  audit?: string;
+  enforce?: string;
+}
+
+interface Props {
+  policies: CustomPolicy[];
+}
+
+export const PolicyTable: FC<Props> = ({ policies }) => {
+  const { isFlagEnabled } = useFeatureFlags();
+
+  policies.forEach((policy) => {
```

Comment:
[POV]: It's not recommended to use `forEach` here as it's a mutator method which in this situation you're modifying the element and that might leads to unexpected behavior so using `map` instead would be recommended

Reply:
good point